python_modules/dagster/dagster_tests/definitions_tests/auto_materialize_tests/asset_daemon_scenario.py

```diff
@@ -206,33 +209,17 @@ class MultiAssetSpec(NamedTuple):
     can_subset: bool = False
 
 
-class AssetDaemonScenarioState(NamedTuple):
-    """Specifies the state of a given AssetDaemonScenario. This state can be modified by changing
-    the set of asset definitions it contains, executing runs, updating the time, evaluating ticks, etc.
-
-    At any point in time, assertions can be made about the state of the scenario. Typically, you
-    would add runs to the scenario, evaluate a tick, then make assertions about the runs that were
-    requested for that tick, or the evaluations that were stored for each asset.
-
-    Args:
-        asset_specs (Sequence[AssetSpec]): The specs describing all assets that are part of this
-            scenario.
-        current_time (datetime): The current time of the scenario.
+@dataclass(frozen=True)
+class AssetScenarioState:
+    """Generic base class for an object which can be used to specify relevent state when simulating
+    an interaction that mutates state involving assets (e.g. their definitions or the state of
+    the event log).
     """
 
     asset_specs: Sequence[Union[AssetSpec, AssetSpecWithPartitionsDef, MultiAssetSpec]]
```

Comment:
This predates this diff, but I don't think the `AssetSpecWithPartitionsDef` and `MultiAssetSpec` make much sense.